client/lib/products-values/index.js

```diff
@@ -359,6 +364,25 @@ export function getDomain( product ) {
 	return product.meta;
 }
 
+export function getJetpackProductDisplayName( product ) {
+	product = formatProduct( product );
+	assertValidProduct( product );
+
+	return JETPACK_PRODUCT_DISPLAY_NAMES?.[ product?.productSlug ];
+}
+
+export function getJetpackProductTagLine( product ) {
+	product = formatProduct( product );
+	assertValidProduct( product );
+
+	return JETPACK_PRODUCT_TAG_LINES?.[ product?.productSlug ];
+}
+
+export function isProductExpiring( product ) {
```

Comment:
I don't think this helper makes a whole lot of sense in this file. It feels it should be defined in the component that consumes it. There is also another reason to move it there: the `moment` usage. 

I suggest that you move this as a method to the component, and wrap that component with the `withLocalizedMoment` HoC if it's a `React.Component`, and with `useLocalizedMoment` hook if it's a functional component.

Reply:
Sounds good. I'll move it to the component and use Hoc there.